client/web-sveltekit/package.json

```
@@ -0,0 +1,42 @@
+{
+	"name": "web-sveltekit",
```

To be consistent with other packages

```suggestion
	"name": "@sourcegraph/web-sveltekit",
```